### PR TITLE
addresses issue #97

### PR DESCRIPTION
**Pull request by BenoistP**
Moves api urls to (.)ENV
set urls to new api.realtoken.community domain (previously api.realt.community)

### CODE REVIEW

**Review by NandyBa — CHANGES_REQUESTED** [review]
Please also update .env.sample

**Comment by NandyBa** [comment]
Why moving url to .env ?

**Comment by BenoistP** [comment]
moving to .env according to @Sigri44 : domain SHOULD not move but ...

**Review by NandyBa — CHANGES_REQUESTED** [review]

File: .env.sample [hunk 1]
```diff
@@ -1,3 +1,5 @@
 BUILD_ENV=
 COMMUNITY_API_KEY=
 THEGRAPH_API_KEY=
+NEXT_PUBLIC_REALTOKENAPI=
+NEXT_PUBLIC_REALTOKENAPI_HISTORY=
```
[the remark this answers is not in the document]
Comment (BenoistP) [reply .env.sample]: right

**Review by NandyBa — DISMISSED** [review]
All works.
I just recommend to remove the prefix `NEXT_PUBLIC_`from the new keys

**Comment by BenoistP** [comment]
done !

**Review by NandyBa — APPROVED** [review]

**Pull request merged**